README.md

```diff
@@ -644,6 +644,24 @@ ort {
 
 To enable this provider, pass `-a VulnerableCode` on the command line.
 
+## OSV
```

Comment:
Commit message: No dot at the end of the title. And we usually just use "README" as the prefix here.

Reply:
In general, this could probably squashed with the previous commit. It seems a bit overly detailed to split these changes.

Reply:
IIRC that's quite common practice in past PRs in ORT, so I prefer to just leave this as-is.